stacks/container-apps/sphinx-nlb.yml

```diff
@@ -474,7 +474,7 @@ Resources:
           TargetGroupArn: !Ref NLBSphinxTargetGroup
       # Role: !Ref ECSServiceIAMRole
       NetworkConfiguration:
-        AwsvpcConfiguration:
+        AwsVpcConfiguration:
```

Comment:
This either changed, or cfn-lint was allowing both and now isn't.

Reply:
That seems to be right now though
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-service-networkconfiguration.html